DiceKeys/Views/DerivationRecipes/DerivationRecipeMenu.swift

```diff
@@ -9,7 +9,7 @@ import SwiftUI
 
 enum DerivationRecipeBuilderType: Equatable {
     case recipe(DerivationRecipe)
-    case template(DerivationRecipeTemplate)
+    case template(DerivationRecipe)
```

Comment:
What is the difference between recipe and template enum, since we are using the same object for both?

Reply:
A recipe is immutable whereas a template can be modified.